webshare/share-cancel-manual.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share cancelled by user</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        setupManualShareTest(true);
+
+        promise_test(t => {
+          return callWhenButtonClicked(() => promise_rejects(
+              t, 'AbortError',
+              navigator.share({title: 'the title', text: 'the message', url: 'data:the url'})));
```

Comment:
Fixed. (It was throwing an exception from an asynchronous context that wasn't being caught by the test runner.)